extensions/ConfirmEdit/Captcha.php

```diff
@@ -604,11 +605,11 @@ public function APIGetAllowedParams( &$module, &$params ) {
 	}
 
 	/**
-	 * @param $module ApiBae
```

Comment:
https://www.theguardian.com/culture/shortcuts/2017/aug/02/from-bae-to-millennial-11-words-that-make-the-heart-sink
>*Bae* Definition of the sort of neologism you probably should have looked up by now, but somehow could never be arsed. You secretly suspect that even if you did, the meaning would simply glide past the eyes in a seizure of inanity.